### PR TITLE
fix(render): use template version

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to https://github.com/camunda/camunda-modeler/issues/2920

In the properties panel header, it now uses the template version as well to retrieve the applied template. 

**Camunda Platform 8**

image

**Camunda Platform 7**

image

### CODE REVIEW

File: src/render/PanelHeaderProvider.js [hunk 1]
```diff
@@ -185,7 +185,8 @@ function getTemplatesService() {
 
 function getTemplate(element, elementTemplates) {
   const templateId = elementTemplates._getTemplateId(element);
-  return templateId && elementTemplates.get(templateId);
+  const templateVersion = elementTemplates._getTemplateVersion(element);
+  return templateId && elementTemplates.get(templateId, templateVersion);
```
Comment (marstamm) [inline src/render/PanelHeaderProvider.js:189]: Why not use `elementTemplates.get(element)` instead? It supports elements as arguments

https://github.com/bpmn-io/bpmn-js-properties-panel/blob/d96930011cec7e5db04776a375fc0e4f41544878/src/provider/element-templates/ElementTemplates.js#L26-L33

Comment (pinussilvestrus) [reply src/render/PanelHeaderProvider.js:189]: Good catch!

Comment (pinussilvestrus) [reply src/render/PanelHeaderProvider.js]: I adjusted it accordingly 👍  Thanks!

File: src/render/PanelHeaderProvider.js [hunk 2]
```diff
@@ -185,7 +185,7 @@ function getTemplatesService() {
 
 function getTemplate(element, elementTemplates) {
   const templateId = elementTemplates._getTemplateId(element);
```
Comment (marstamm) [inline src/render/PanelHeaderProvider.js:187]:
```suggestion
```

File: src/render/PanelHeaderProvider.js [hunk 3]
```diff
@@ -185,7 +185,7 @@ function getTemplatesService() {
...
-  return templateId && elementTemplates.get(templateId);
+  return templateId && elementTemplates.get(element);
```
Comment (marstamm) [inline src/render/PanelHeaderProvider.js:188]:
```suggestion
  return elementTemplates.get(element);
```

**Review by marstamm — COMMENTED** [review]
It can be simplified further: `elementTemplates.get` returns `null` when no template is set

File: src/render/PanelHeaderProvider.js (hunk 3 above)
Comment (pinussilvestrus) [reply src/render/PanelHeaderProvider.js:188]: That's true as we anyway check for `template && ...` anywhere we use this helper.

**Pull request merged**